cmd/sansshell-server/main.go

```diff
@@ -127,11 +127,13 @@ func main() {
 	policy := util.ChoosePolicy(logger, defaultPolicy, *policyFlag, *policyFile)
 	ctx := logr.NewContext(context.Background(), logger)
 
+	parsed, err := opa.NewAuthzPolicy(ctx, policy, opa.WithDenialHintsQuery("data.sansshell.authz.denial_hints"))
+	if err != nil {
+		log.Fatalf("Invalid policy: %v\n", err)
+	}
+
 	if *validate {
-		_, err := opa.NewAuthzPolicy(ctx, policy)
-		if err != nil {
-			log.Fatalf("Invalid policy: %v\n", err)
-		}
+
 		fmt.Println("Policy passes.")
```

Comment:
is this still necessary?

Reply:
"run as a one-shot and exit after parsing" was pre-existing, so I left it in to preserve that behavior. We use it in https://github.com/Snowflake-Labs/sansshell/blob/07514978b337858cc74ecf46df814c78610bf4dc/testing/integrate.sh#L466-472 as part of our integration testing.